package.json

```diff
@@ -4,18 +4,20 @@
   "description": "Component to handle the main photo gallery of the checkout page.",
   "main": "carousel.app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
     "start": "webpack -d -w",
```

Comment:
@aku2nnya You can disregard my earlier comments about splitting these out! I see that it is done here. One small recommendation: I would try to align the build and build-watch commands to be very similar so that anyone using your package in the future understands they are similar (e.g., npm run build and npm run build:watch, etc.!).

It may be confusing to a user to have start run webpack, as that usually starts the program (e.g., the server in our case!).

Reply:
Sounds good, made the changes, thanks